idl/floatingBaseEstimatorRPC/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2016 Istituto Italiano di Tecnologia  iCub Facility
+# Authors: Silvio Traversaro
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+yarp_idl_to_dir(codyco/floatingBaseEstimatorRPC.thrift ${CMAKE_CURRENT_SOURCE_DIR}/autogenerated WBDIDLSERVER_SRC WBDIDLSERVER_HEADERS WBDIDLSERVER_INCLUDES)
```

Comment:
Are we sure this will not cause any problems?

Reply:
See https://github.com/robotology/whole-body-estimators/pull/16#discussion_r380174507 .